service/spi/src/main/java/gov/va/vro/service/spi/model/AssessmentResult.java

```diff
@@ -0,0 +1,33 @@
+package gov.va.vro.service.spi.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Map;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class AssessmentResult {
```

Comment:
This is essentially copy of FullHealthDataAssessmentResponse except it is missing ABDEvidence. That makes sense since that is the result of Health Assessment. Can't we just use that one here? Maybe by including api project?